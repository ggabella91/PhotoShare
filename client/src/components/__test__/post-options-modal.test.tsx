import { shallow } from 'enzyme';
import React from 'react';
import PostOptionsModal from '../post-options-modal/post-options-modal.component';

it('renders a post options modal', () => {
  const postOptionsModalWrapper = shallow(
    <PostOptionsModal show={true} onHide={() => {}} archive={() => {}} />
  );

  expect(postOptionsModalWrapper).toMatchSnapshot();
});
