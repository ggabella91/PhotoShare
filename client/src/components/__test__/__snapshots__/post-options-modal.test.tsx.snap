// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a post options modal 1`] = `
<Modal
  animation={false}
  autoFocus={true}
  backdrop={true}
  centered={true}
  dialogAs={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "render": [Function],
    }
  }
  dialogClassName="post-options-modal"
  enforceFocus={true}
  keyboard={true}
  onHide={[Function]}
  restoreFocus={true}
  show={true}
>
  <ModalBody
    className="post-options-modal-body"
  >
    <div
      className="archive"
      onClick={[Function]}
    >
      <span>
        Archive
      </span>
    </div>
    <div
      className="cancel"
      onClick={[Function]}
    >
      <span>
        Cancel
      </span>
    </div>
  </ModalBody>
</Modal>
`;
